Formula/hive.rb

```diff
@@ -29,6 +29,7 @@ def caveats; <<-EOS.undent
   end
 
   test do
+    system "#{bin}/schematool -initSchema -dbType derby"
```

Comment:
Last thing: please make this: `system "#{bin}/schematool", "-initSchema", "-dbType", "derby"`